pkg/templates/git.go

```diff
@@ -158,7 +163,8 @@ func syncTemplateFromRef(
 			Name:       entity.Name,
 			Version:    ref,
 			Source:     source + "?ref=" + repo.Reference,
-			Schema:     *schema,
+			Schema:     *originSchema,
+			UiSchema:   fileSchema.Expose(),
```

Comment:
What happen when we revert uiSchema to the default?

Reply:
The reset logic before will reset all template version and UI schema. Using template version bus to revert to default.